core/base/src/TColor.cxx

```diff
@@ -361,11 +361,11 @@ TColor::TColor(const TColor &color) : TNamed(color)
 /// This method should be called before the ApplicationImp is created (which
 /// initializes the GUI colors).
 
-void TColor::InitializeColors()
+void TColor::InitializeColors(Bool_t force)
```

Comment:
It is not obvious what is the use case for re-initializing the TColor.   Could you add a comment giving some context?